### DISCUSSION TITLE
Three drives murdered by chia in less than one month

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#6040] [General]
Describe the bug
DEV here will call this a 'feature' but IMHO killing drives is still murder.

To Reproduce

Just run chia as advertised.

Expected behavior
I expected what is marketed, I didn't expect to kill brand new drives.

Screenshots

Desktop (please complete the following information):

OS: [e.g. Linux]
all hw&sw same-same drives die
Additional context

first drive killed was an NMVE samsung evo pro 980, 1TB died after 100 plots ( 2weeks about one month ) ago, 150TBW, 25% of warranty of 600TBW, useless now; I still have a stock of NVME's, when they're gone I will use the mp600 corsairs that I also have
2.) second drive just for fun WD 1TB blue I did a 'chia plots create' one time, and it went on 12+ hours, then smoked the drive, the internal coils are burned out. ( this my fault, u hear from the nimrods how you can use normal drives to plot, just one time I wanted to test that assertion )

3.) third drive just yesterday, trying to get the 'farmer' working the 'chia farm summary' always has all zeros, so on a clean machine and synced, I did everything possible to get the farmer to farm 51 plots on an attached drive, the logs said to run 'chia plots check', first I ran with '5', it kept telling me 30, during this time HPOOL went offline on these plots, to which today I stopped all 'chia processes' and found out that the entire HDD had lost its zero-block, and was now unrecognizable to the system, I had to run 'foremost' to recover the 'ext-f' format, now the drive is unusable, brand-new WD purple 6TB with 51 plots, what interesting is the farmer always said run 'chia plots check', why in the hell, I mean YOUR SOFTWARE made the plots, why must they be check 1000's of times, after trillions of writes to make them.

Everyone hates CHIA, every programmer has gone nutz, as chia-dev is the most incompentent bunch of nim-rods in the worlds history, SW now means shit-ware, and it be synonymous with chia-dev forever.

Green my arse, its like the 'disk manufacturing' got together and said 'how can we do the gpu thingy with hdd's?', well its done, chia has invented a means to drive up hdd prices 10x and failures 10x as well. So your 5yr warranty is 2 weeks, the manufacturers are already banning mining on hdd, ssd, nvme ...

**Comment by XIII39** [comment]
I don't hate Chia... yet.. 
Making 300 plots per day, currently at 3,300 plots my 2TB WD Black SN750 SSDs, harvesting on Seagate Exos X16 SAS, WD Gold and Red Pro HDDs. No issues, no dead drives... Yet..

Mind you, I don't run my drives at 70+ degrees Celsius
I've also heard (it's just hearsay) but if you live in the United States you generally get the largest percentage of dud hardware because you'll just go out any buy a new one.. But that's probably just nonsense. 

Lastly, WD Purple are known for shitty writes.. That drive is designed for surveillance cameras which don't necessary need complete data to function. I haven't personally tested them for their file stability but that's what I've heard over the years. WD Blue, well.. That's just a given.. It's the iPhone of HDDs.. Garbage. It can barely hold an operating system used for just checking email yet alone massive writes..

Chia has its problems but so does some of the hardware choices that people are making.
Let's all get rich on a coin but buy Walmart brand hardware then blame everything except our bad (cheap) choices. 

Race a McLaren with a Pinto.. is it the car, the driver or the road that makes the Pinto lose?

**Comment by chiabug** [comment]
LOL.  I can understand how such an experience can make you hate Chia.  To be sure the Chia software is far from polished.  While it may be "green" compared to some of the other coin mining, at the moment it is far from green when it comes to the amount of hardware it is sending to the landfill.  Hopefully this will get better with time.

Btw I am one of the nimrods who plots using regular drives.  I haven't had an issue with the drives.  One of my Pinto's does have an electronic smell coming from the CPU after plotting but so far it is still in the race somewhere way behind the McLaren.
